crates/sui-json-rpc/src/read_api.rs

```diff
@@ -97,6 +131,21 @@ impl RpcReadApiServer for ReadApi {
             .try_into()?)
     }
 
+    async fn get_dynamic_field_object(
```

Comment:
In https://docs.sui.io/devnet/build/programming-with-objects/ch5-dynamic-fields, we use `dynamic field` and `dynamic object field` to refer to two different types of dynamic fields. Does this endpoint support both types?

Reply:
yes both are supported.
https://github.com/MystenLabs/sui/blob/3735554f1b9f0cbf1711ce245bb74d54b73a5c8b/crates/sui-types/src/dynamic_field.rs#L19
https://github.com/MystenLabs/sui/blob/3735554f1b9f0cbf1711ce245bb74d54b73a5c8b/crates/sui-types/src/dynamic_field.rs#L27-L33

The `DynamicFieldType` enum is used to indicate whether it is dynamic field or dynamic field object.